public/kibi_timeline.js

```diff
@@ -80,6 +81,23 @@ define(function (require) {
         if (!timeline) {
           // create a new one
           timeline = new vis.Timeline($element[0]);
+          var timezone = config.get('dateFormat:tz');
+          if (timezone !== 'Browser') {
+            var utcOffset = null;
+            if (timezone.startsWith('Etc/GMT')) {
+              // with this line we are converting 'Etc/GMT +/- [hours] into '+/- [hours]:00'
```

Comment:
this whole method should be extracted and moved into timelineHelper 
and please there should be a few tests for this method